src/controllers/rest/impl/security/AdminController.ts

```diff
@@ -8,13 +8,17 @@ import { BodyParams } from "@tsed/platform-params";
 import { StatusCodes } from "http-status-codes";
 import { Authorize } from "@tsed/passport";
 import { IAdminController } from "../../IAdminController.js";
+import { IpBlackListRepo } from "../../../../db/repo/IpBlackListRepo.js";
 
 @Hidden()
 @Authorize("loginAuthProvider")
 @Controller("/admin")
 export class AdminController extends AbstractAdminController implements IAdminController {
-    public constructor(@Inject() private UserAdminService: UserAdminService) {
-        super(UserAdminService);
+    public constructor(
+        @Inject() private UserAdminService: UserAdminService,
+        @Inject() private BlackListRepo: IpBlackListRepo,
```

Comment:
same as above, but also, this does not need `private`

Reply:
Done